NorimaConsulting/training-and-onboarding
# REC-22 - Machine Setup Check

## Resolves: Possible formatting and OS conflicts.
[Jira REC-22](https://norima-alpha.atlassian.net/jira/software/projects/REC/boards/142/backlog?selectedIssue=REC-22).

1. [Followed machine setup instructions](https://github.com/NorimaConsulting/training-and-onboarding/blob/winter2022/main/docs/machine_setup_get_started_task.md).
3. Installed pnpm and ESLint. 
4. Fixed Eol (End of line) issue with the team. 
5. Added Signe.tsx, SpinButton with two small functions in app to test ESLint, and get some practice with v6 React Router Dom. 

## Testing: 
1. Go to: http://localhost:3000/created_by  and in terminal run **pnpm dev** or  **npm start**
2. See names: James and Signe.
3. Click on each name to see a different comic sound effect word appear below and see the URL/route change. 
*ex/ http://localhost:3000/created_by/yellow-power-ranger/Signe*

## Screen Shot: 
<img width="570" alt="image" src="https://user-images.githubusercontent.com/83307690/149037091-50a62db6-584c-416f-8f99-1f5a1e909d9a.png">
